website/content/docs/v0.10/Guides/disaster-recovery.md

```diff
@@ -0,0 +1,145 @@
+---
+title: "Disaster Recovery"
+description: "Procedure for snapshotting etcd database and recovering from catastrophic control plane failure."
+---
+
+`etcd` database backs Kubernetes control plane state, so if `etcd` service is unavailable
+Kubernetes control plane goes down, and the cluster is not recoverable until `etcd` is recovered with contents.
+`etcd` consistency model builds around consensus protocol Raft, so for highly-available control plane clusters,
+loss of one control plane doesn't impact cluster health.
+In general, `etcd` stays up as long as quorum nodes are up.
+For three control plane node Talos cluster, this means that the cluster tolerates a failure of any single node,
+but losing more than one node leads to complete cluster failure.
+Because of that, it is important to take backups of `etcd` state to have a snapshot to recover cluster from
+in case of catastrophic failure.
+
+## Backup
+
+### Snapshotting `etcd` Database
+
+Create a consistent snapshot of `etcd` database with `talosctl etcd snapshot` command:
+
+```bash
+$ talosctl -n <IP> etcd snapshot db.snapshot
+etcd snapshot saved to "db.snapshot" (2015264 bytes)
+snapshot info: hash c25fd181, revision 4193, total keys 1287, total size 3035136
+```
+
+This database snapshot can be taken on any healthy control plane node, as all `etcd` instances contain
```

Comment:
Might add a note that the IP above must be the IP address of one of the controlplane node

Reply:
and that the filename `db.snapshot` is arbitrary